.github/ISSUE_TEMPLATE/suggest_wallet.yaml

```diff
@@ -48,13 +48,6 @@ body:
       description: Please provide a URL (e.g. to the website of the wallet).
     validations:
       required: true
-  - type: input
-    id: wallet_live_date
-    attributes:
-      label: When did the wallet go live to users?
-      description: Please provide a date when the wallet was usable by the public
-    validations:
-      required: true
```

Comment:
@nhsz I think we still need this as part of the initial issue request... Our [listing policy](https://ethereum.org/en/contributing/adding-wallets/#the-must-haves) requires the product be live for 6 months, and this would be how we check that.

Shouldn't have any impact on the other changes